libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -258,6 +258,9 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
 // constructor
 AP_GPS::AP_GPS()
 {
+    static_assert((sizeof(_initialisation_blob) * 9) < (4800 * GPS_BAUD_TIME_MS * 1e-3),
```

Comment:
8 bits to a byte + stop byte. It's the actual cost in terms of baud over the wire for an 8N1 uart... This is what I really debated using #define's for, but we only need it in the one spot.